src/livecodes/UI/deploy.ts

```diff
@@ -139,6 +139,7 @@ export const createDeployUI = async ({
       deps: { getLanguageExtension: deps.getLanguageExtension },
     }).catch((error: any) => {
       if (error.message === 'Repo name already exists') {
+        // TODO: i18n this
```

Comment:
What about error messages i18n?

Reply:
Yes, I think we should translate this.
In this specific case we already know the error message so we can just translate it.